src/main/java/life/qbic/io/commandline/CommandLineParser.java

```diff
@@ -0,0 +1,50 @@
+package life.qbic.io.commandline;
+
+import java.io.IOException;
+import life.qbic.io.parser.IdentifierParser;
+import picocli.CommandLine;
+
+public class CommandLineParser {
+
+  /**
+   * parses all passed CLI parameters Prints help menu if no commandline parameters were passed
+   * verifies whether all mandatory commandline parameters have been passed (IDs and username)
+   *
+   * <p>also reads the provided identifiers if they were passed as a file
```

Comment:
```suggestion
   * also reads the provided identifiers (as if)/(when) they were passed as a file
```
I am not sure as to what is meant here.

Reply:
Why not use a library for command-line parsing here?

Reply:
> I am not sure as to what is meant here.

Still not sure.

Reply:
It is irritating. I removed this statement. No matter if you pride a list of ids in the command line or in a file, the access to the ids will be over ONE property.